### PR TITLE
Some translation made in README-en.md

### PR DESCRIPTION
**Pull request by RaytorHu**
Hi, I voluntary made some translation to the posts from March 28 to March 21. Can you accept my pull request? Hope this helps.

### CODE REVIEW

File: README-en.md [hunk 1]
```diff
@@ -52,32 +52,32 @@ If you are unsure, please contact me.
 ### March 21, 2018
 
 #### Airing - [Github](https://github.com/airingursb), [Blog](http://ursb.me)
-- :white_check_mark: [四时](https://itunes.apple.com/us/app/%E5%9B%9B%E6%97%B6/id1272513774?l=zh&ls=1&mt=8)：Shiny Weather App - [更多介绍](https://github.com/airingursb/4times-front-end)
-- :clock8: [双生](https://github.com/oh-bear/2life)：遇见另一半的美好（共享日记）
+- :white_check_mark: [四时](https://itunes.apple.com/us/app/%E5%9B%9B%E6%97%B6/id1272513774?l=zh&ls=1&mt=8)：Shiny Weather App - [More Details](https://github.com/airingursb/4times-front-end)
+- :clock8: [双生](https://github.com/oh-bear/2life)：The beauty of meeting the other half（Shared Diary）
```
Comment (leon332157) [inline README-en.md]: The beauty of meeting the other half of your life（Shared Diary）

File: README-en.md [hunk 2]
```diff
@@ -52,32 +52,32 @@ If you are unsure, please contact me.
...
 
 #### Arczzir - [Github](https://github.com/arczzir)
-* :white_check_mark: [Jed](https://itunes.apple.com/cn/app/jed/id1234853584)：macOS JSON file editor - [更多介绍](https://www.bilibili.com/video/av10147282/)
+* :white_check_mark: [Jed](https://itunes.apple.com/cn/app/jed/id1234853584)：macOS JSON file editor - [More Details](https://www.bilibili.com/video/av10147282/)
 
 #### Drinking - [Github](https://github.com/drinking), [Blog](http://drinking.github.io/)
-* :white_check_mark: [饭起](https://itunes.apple.com/cn/app/%E9%A5%AD%E8%B5%B7-%E5%85%B3%E4%BA%8E%E9%A3%9F%E7%89%A9%E7%9A%84%E7%88%B1%E4%B8%8E%E6%95%85%E4%BA%8B/id1209331941?mt=8)：关于食物的爱与故事  - [更多介绍](http://fancymeet.com/)
+* :white_check_mark: [饭起](https://itunes.apple.com/cn/app/%E9%A5%AD%E8%B5%B7-%E5%85%B3%E4%BA%8E%E9%A3%9F%E7%89%A9%E7%9A%84%E7%88%B1%E4%B8%8E%E6%95%85%E4%BA%8B/id1209331941?mt=8)：Stories and love of food  - [More Details](http://fancymeet.com/)
```
Comment (leon332157) [inline README-en.md]: Stories for food and love

File: README-en.md [hunk 3]
```diff
@@ -52,32 +52,32 @@ If you are unsure, please contact me.
...
 
-#### Yuuta - [Github](https://github.com/Trumeet), [网站](https://yuuta.moe)
-* :white_check_mark: [Dir](https://coolapk.com/apk/kh.android.dir)：a simple and elegant Android trash file clean tool, prevent file from re-generate - [更多介绍](https://dir.yuuta.moe/zh/)  
+#### Yuuta - [Github](https://github.com/Trumeet), [Website](https://yuuta.moe)
+* :white_check_mark: [Dir](https://coolapk.com/apk/kh.android.dir)：a simple and elegant Android trash file clean tool, prevent file from re-generate - [More Details](https://dir.yuuta.moe/zh/)  
 
-#### 曦莫琅 - [Github](https://github.com/ximolang), [博客](http://www.txliang.com/)
-* :white_check_mark: [诗说社](http://shishuo.wesnice.com/)：一个文青、情怀人群的聚集地，原创诗词的发布平台
+#### 曦莫琅 - [Github](https://github.com/ximolang), [Blog](http://www.txliang.com/)
```
Comment (leon332157) [inline README-en.md]: A platform for young artists and well learned man to share their poetries.

File: README-en.md [hunk 4]
```diff
@@ -52,32 +52,32 @@ If you are unsure, please contact me.
...
 #### ruzhan123 - [Github](https://github.com/ruzhan123)
-* :white_check_mark: [Awaker](https://www.coolapk.com/apk/155953)：科幻，地理阅读杂志的 Android 应用 - [更多介绍](https://github.com/ruzhan123/awaker)
+* :white_check_mark: [Awaker](https://www.coolapk.com/apk/155953)：An Android app for science fictions and geographical magazines - [More Details](https://github.com/ruzhan123/awaker)
 
 #### cwang22 - [Github](https://github.com/cwang22), [Blog](https://seewang.me)
 * :white_check_mark: [Buy All Steam Games](http://steam.seewang.me)：How much you money do you need to buy all Stream games?
 
 #### santa - [Github](https://github.com/santa-cat)
-* :white_check_mark: [一天](https://fir.im/oneday)：24小时结识朋友的社交应用 (Android、iOS)
+* :white_check_mark: [一天](https://fir.im/oneday)：A social app for making friends in 24hr (Android、iOS)
```
Comment (leon332157) [inline README-en.md]: A social app for makingn new friends 24/7 (Android、iOS)

File: README-en.md [hunk 5]
```diff
@@ -52,32 +52,32 @@ If you are unsure, please contact me.
...
 
 #### Caij - [Github](https://github.com/Caij)
-* :white_check_mark: [EMore](https://www.coolapk.com/apk/com.caij.emore)：轻量简单的第三方微博客户端 
+* :white_check_mark: [EMore](https://www.coolapk.com/apk/com.caij.emore)：A simple and light-weight, third-party client application of weibo.
```
Comment (leon332157) [inline README-en.md]: A simple and light-weight, third-party client for Weibo.

File: README-en.md (hunk 4 above)
Comment (leon332157) [reply README-en.md]: making

**Comment by 1c7** [comment]
@RaytorHu Hi! Thanks!
@leon332157 Thanks to you too!

**Comment by 1c7** [comment]
@leon332157  review 的都挺好的，谢谢，但我最近没时间，就不把你的改进版翻译复制粘贴了，等有空了回来看看要不要根据你的提议改改，现在先 merge 了

**Comment by leon332157** [comment]
@1c7 没事，我有空复制一下再开pr吧

**Pull request merged**